docs/ROADMAP.md

```diff
@@ -7,7 +7,7 @@ All the dates and the scope are aspirational.
 To discuss the roadmap and the related initiatives, use the
 `#declarative-gradle` channel on the [Gradle Community Slack](https://gradle.org/slack-invite)
 
-## First Early Access Preview (EAP)
+## First Early Access Preview (EAP) - July 2024
```

Comment:
NIT in this context, but it is better to not change the headers as it is used to generate anchor URL. 
I would recommend adding a list of milestones in the top of the page, and keep the target dates there